ecommerce/mail_api.py

```diff
@@ -30,11 +40,23 @@ def get_bulk_enroll_email_context(product_coupon):
         .order_by("-created_on")
         .first()
     )
-    return {
-        "enrollable_title": product_coupon.product.content_object.title,
+    product_object = product_coupon.product.content_object
+    context = {
+        "enrollable_title": product_object.title,
         "enrollment_url": enrollment_url,
         "company_name": company_name,
     }
+    return (
+        recipient,
+        context,
+        api.EmailMetadata(
+            tags=[BULK_ENROLLMENT_EMAIL_TAG],
+            user_variables={
+                "enrollment_code": product_coupon.coupon.coupon_code,
+                product_coupon.product.content_type.model: product_object.text_id,
```

Comment:
Use `product_coupon.product.type_string` instead of `product_coupon.product.content_type.model`?